docs/rules/jsx-no-undef.md

```diff
@@ -18,6 +18,52 @@ var Hello = require('./Hello');
 <Hello name="John" />;
 ```
 
+## Rule Options
+
+```js
+...
+"jsx-no-undef": [<enabled>, { "importedOnly": <boolean> }]
```

Comment:
nit: In #1071 we are adding `react/` to lines like this. It would be nice to get that in here if we touch this again.